### ISSUE TITLE
Loss of resolution due to using 10 minute gusts

### ISSUE THREAD

**Issue by uajqq** [grebleem/WeatherLinkliveUDP#6]
I have my weewx archive interval set to the default 5 min. I noticed my wind gust graphs all had "blunted" tops as compared to the more jagged peaks I was seeing with my old station. The issue is that the live UDP updates contain only 10-minute gust info, so every time there is a new highest gust it persists for the next 2 weewx archive cycles.

To try to improve resolution, I changed the driver so the `gust` parameter in the packet is updated only by the `wind_speed_hi_last_2_min` data field in the HTTP packet, while the UDP packet updates instantaneous wind speed and direction only.

The downside is that it looks like the console only updates the 2-minute gust value every 60 seconds, even if queried every 10 seconds. That results in a lag up to 60 seconds in the reported gust speed, so sometimes I see a higher wind speed than the listed gust speed for a few seconds. So the graphs are more accurate with better resolution, but the live display is very occasionally out of sync.

I made the change in the code just after sunset--you can see the change in the resolution of the gust chart:
![image](https://user-images.githubusercontent.com/46248396/87226354-23515c00-c361-11ea-98cb-043a9a072302.png)

**Comment by grebleem** [comment]
Hi, That are some very interesting thoughts! I have been struggling whit this as well. The reason that there is a 10 min gust value in de UPD, is that Davis is using it in the phone app, but using this 10 minutes gust value in the database is 'less' accurate.
The HTTP request in my setup is set at 10 seconds, so the live gust update should be updated every 10 seconds. Just one question, why use the 2-minute and not use the 1-minute gust information?

I will do some testing as well (unfortunately not much wind today, but I will keep this running for at least a week).

To summary: 
UDP-> No gust recording at all, only `wind_speed_last` and `wind_dir_last`.
HTTP -> `wind_speed_last`,`wind_dir_last`, And `wind_speed_avg_last_1_min` and  `wind_dir_scalar_avg_last_1_min`.

Thanks for this great input!

**Comment by grebleem** [comment]
I see now why you suggested the 2-minute. The 1-minute is an average and not a high value.

**Comment by grebleem** [comment]
https://github.com/grebleem/weewx-weatherlinkliveudp/tree/development

**Comment by hoevenvd** [comment]
Yes, same here. At 1800 today, I've implemented the driver. Before 1800 I did use the Vantage2-driver (talking to my Weatherlink IP). You see a not so smooth windgust chart starting at 1800.

I've switched now to the development-driver.

![2020-07-12](https://user-images.githubusercontent.com/2419916/87254902-84bf1b00-c486-11ea-89d7-06123267947a.png)

**Comment by uajqq** [comment]
Good news on this front:
![IMG_0668](https://user-images.githubusercontent.com/46248396/87586172-25435400-c6ae-11ea-8749-7bbfcc35a130.jpeg)

**Comment by grebleem** [comment]
Nice, is this information from the developers of the wll?

**Comment by uajqq** [comment]
Yeah, as it turns out they have a Gitter chatroom they’re pretty active in: [https://gitter.im/WeatherLink/weatherlink-live-local-api](url)
